tests/fastcs/panda/test_hdf_panda.py

```diff
@@ -149,7 +149,7 @@ def flying_plan():
             "uid": ANY,
             "data_key": data_key_name,
             "mimetype": "application/x-hdf5",
-            "uri": "file://localhost" + str(tmp_path / "test-panda.h5"),
+            "uri": "file://localhost/" + str(tmp_path / "test-panda.h5").lstrip("/"),
```

Comment:
What does one of these windows paths actually look like? Not sure what the right `file://` semantics are for drive letter etc. What does a web browser show if you point to `file://something_on_c_drive` on windows?

Reply:
Web browsers (at least firefox, chrome and edge) do correctly understand a path like `file:///C:\Instrument\Var\logs\ioc\ARINST-20241030.log`. Forward slashes rather than backslashes would also work - but the drive letter part is what it wants as far as i understand it.